src/OpenTelemetry.Shims.OpenTracing/TracerShim.cs

```diff
@@ -83,7 +83,9 @@ IEnumerable<string> GetCarrierKeyValue(Dictionary<string, IEnumerable<string>> s
                 spanContext = this.textFormat.Extract(carrierMap, GetCarrierKeyValue);
             }
 
-            return !spanContext.IsValid ? null : new SpanContextShim(spanContext);
+            // TODO: remove comment after spanshim changes
+            // return !spanContext.IsValid ? null : new SpanContextShim(spanContext);
```

Comment:
this can be done now itself right?

Reply:
After span shim PR merge we can enable this one.